readme.md

```diff
@@ -1,3 +1,21 @@
 # Design System Team Labels
 
 We use this repo to synchronise the labels we use across our repositories.
+
+It uses [github-label-sync](https://github.com/Financial-Times/github-label-sync).
+
+## Updating labels
+
+Labels are configured in `config.json`.
```

Comment:
We could add that once the PR with label changes is merged to master the script will run to update all the repos.